### ISSUE TITLE
Spellcheck

### ISSUE THREAD

**Issue by ryanhugh** [ryanhugh/searchneu#9]
Currently, if a user misspells a search query there is a good chance nothing will come up. If we could add support to automatically detect and fix spelling mistakes (like Google and Bing) that would be great!

**Comment by NEUDitao** [comment]
Seems like #79 would fix this...

**Comment by ryanhugh** [comment]
Yup! And elastic search should have some feature we should be able to use too
